scripts/README.md

```diff
@@ -0,0 +1,16 @@
+# Script usage
```

Comment:
Since this is a README that pertains only to the `sign.sh` file (but is in the generic `scripts/` folder) perhaps rename it to `scripts/sign.sh.README.md` or maybe make this a _section_ in the file (and make it clear where the next author of the next script might add their documentation?).

Reply:
I think the second option was my intention. "Script usage" is meant to be the title of the README (meaning that it is the readme for all scripts in the script directory). The subheading "sign.sh" (on line 3) is meant to be the section that describes the usage for just the sign.sh file, and subsequent sections can be created below or above. Do you have suggestions on how I could improve this format for clarity?

Reply:
Just my $.02 USD. I recommend putting each script into its own markdown file. In README.md, I'd have an abstract for the script and a link to script's markdown. This would make it easier to maintain, in my opinion, since it separates the doc for the specific script from what may grow to be a very large README.